### PR TITLE
add region approximate size

### PR DESCRIPTION
**Pull request by huachaohuang**
Get approximate region size from user properties.

### CODE REVIEW

**Comment by huachaohuang** [comment]
PTAL @siddontang @zhangjinpeng1987

**Comment by siddontang** [comment]
LGTM

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -1435,6 +1435,10 @@ impl Peer {
         None
     }
 
+    pub fn approximate_size(&self) -> Result<u64> {
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/peer.rs]: Don't add the calculation into `raftstore` thread, it might block raftstore thread.

Comment (siddontang) [inline src/raftstore/store/peer.rs]: I think it is ok now, we will do it in another thread later

Comment (siddontang) [inline src/raftstore/store/peer.rs]: I think it is ok now, we will do it in another thread later

Comment (huachaohuang) [inline src/raftstore/store/peer.rs]: Do you mean I should put it in the worker instead of `heartbeat_pd`?

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**